### PR TITLE
Sx26 search plug name

### PR DESCRIPTION
**Pull request by maggie-moheb**
@daliamaarek @Youmna-Salah @ahmad3m3 @AmirMaurice please review

### CODE REVIEW

**Comment by AmirMaurice** [comment]
@maggie-moheb only you need a class diagram :see_no_evil:

**Comment by maggie-moheb** [comment]
@AmirMaurice  done :+1:

**Comment by AmirMaurice** [comment]
@maggie-moheb perfect :+1:

File: SmartX/app/src/main/java/com/smartx/cookies/smartx/CustomGrid.java [hunk 1]
```diff
@@ -70,21 +86,33 @@ public View getView(int position, View convertView, ViewGroup parent) {
...
-            imageView.setImageResource(plugPhotos[position]);
+            textView.setText(plugNames.get(position));
+            imageView.setImageResource(plugPhotos.get(position));
         } else {
             grid = (View) convertView;
         }
-
         return grid;
     }
+
```
Comment (daliamaarek) [inline SmartX/app/src/main/java/com/smartx/cookies/smartx/CustomGrid.java:108]: please add documentation for filter method

**Comment by maggie-moheb** [comment]
@daliamaarek  done :+1:

**Comment by AmirMaurice** [comment]
@maggie test is great :+1:

**Comment by maggie-moheb** [comment]
@Youmna-Salah  done

**Pull request merged**